symbolic-debuginfo/src/dwarf.rs

```diff
@@ -684,25 +717,15 @@ impl<'d, 'a> DwarfUnit<'d, 'a> {
             //
             // XXX: Maybe we should actually parse the ranges in the resolve function and always
             // look at the symbol table based on the start of the DIE range.
-            let symbol_name = if !inline {
-                self.inner
-                    .info
-                    .symbol_map
-                    .lookup_range(function_address..function_end)
-                    .and_then(|symbol| {
-                        symbol
-                            .name
-                            .clone()
-                            .map(|n| Name::new(n, NameMangling::Mangled, self.language))
-                    })
-            } else {
+            let symbol_name = if self.prefer_dwarf_names || inline {
```

Comment:
This is part 2 of the change: Skipping symbol table lookup if DWARF names are preferred.